test/fixtures/failing_test.json

```diff
@@ -0,0 +1,201 @@
+    "contents_url": "https://api.github.com/repos/SimonLab/github_app/contents/{+path}",
+    "compare_url": "https://api.github.com/repos/SimonLab/github_app/compare/{base}...{head}",
+    "merges_url": "https://api.github.com/repos/SimonLab/github_app/merges",
+    "archive_url": "https://api.github.com/repos/SimonLab/github_app/{archive_format}{/ref}",
+    "downloads_url": "https://api.github.com/repos/SimonLab/github_app/downloads",
+    "issues_url": "https://api.github.com/repos/SimonLab/github_app/issues{/number}",
+    "pulls_url": "https://api.github.com/repos/SimonLab/github_app/pulls{/number}",
+    "milestones_url": "https://api.github.com/repos/SimonLab/github_app/milestones{/number}",
+    "notifications_url": "https://api.github.com/repos/SimonLab/github_app/notifications{?since,all,participating}",
+    "labels_url": "https://api.github.com/repos/SimonLab/github_app/labels{/name}",
+    "releases_url": "https://api.github.com/repos/SimonLab/github_app/releases{/id}",
+    "deployments_url": "https://api.github.com/repos/SimonLab/github_app/deployments",
+    "created_at": "2017-05-30T15:26:26Z",
+    "updated_at": "2017-05-30T15:36:26Z",
+    "pushed_at": "2017-06-20T11:21:34Z",
+    "git_url": "git://github.com/SimonLab/github_app.git",
+    "ssh_url": "git@github.com:SimonLab/github_app.git",
+    "clone_url": "https://github.com/SimonLab/github_app.git",
+    "svn_url": "https://github.com/SimonLab/github_app",
+    "homepage": null,
+    "size": 63,
+    "stargazers_count": 0,
+    "watchers_count": 0,
+    "language": "Elixir",
+    "has_issues": true,
+    "has_projects": true,
+    "has_downloads": true,
+    "has_wiki": true,
+    "has_pages": false,
+    "forks_count": 0,
+    "mirror_url": null,
+    "open_issues_count": 7,
+    "forks": 0,
+    "open_issues": 7,
+    "watchers": 0,
+    "default_branch": "initialise"
+  },
+  "sender": {
+    "login": "SimonLab",
+    "id": 6057298,
+    "avatar_url": "https://avatars1.githubusercontent.com/u/6057298?v=3",
+    "gravatar_id": "",
+    "url": "https://api.github.com/users/SimonLab",
+    "html_url": "https://github.com/SimonLab",
+    "followers_url": "https://api.github.com/users/SimonLab/followers",
+    "following_url": "https://api.github.com/users/SimonLab/following{/other_user}",
+    "gists_url": "https://api.github.com/users/SimonLab/gists{/gist_id}",
+    "starred_url": "https://api.github.com/users/SimonLab/starred{/owner}{/repo}",
+    "subscriptions_url": "https://api.github.com/users/SimonLab/subscriptions",
+    "organizations_url": "https://api.github.com/users/SimonLab/orgs",
+    "repos_url": "https://api.github.com/users/SimonLab/repos",
+    "events_url": "https://api.github.com/users/SimonLab/events{/privacy}",
+    "received_events_url": "https://api.github.com/users/SimonLab/received_events",
+    "type": "User",
+    "site_admin": false
+  },
+  "installation": {
+    "id": 31449
+  }
+}
```

Comment:
I think adding these long fixtures isn't the best in terms of test readability, perhaps look at getting rid of some of the noise here so that we can read these files